test/e2e/e2e-helpers.js

```diff
@@ -86,6 +86,24 @@ function expectNavigateAwayFrom(client, urlSubstring) {
     .to.not.contain(urlSubstring).before(Timeouts.slow);
 }
 
+// Expects navigation lands at a path with the given `urlSubstring`.
+function expectNavigateAwayFromExact(client, urlSubstring) {
+  client.expect.element('.js-test-location').attribute('data-location')
+    .to.not.equal(urlSubstring).before(Timeouts.slow);
+}
+
+// Expects navigation lands at a path with the given `urlSubstring`.
+function expectNavigateTo(client, urlSubstring) {
+  client.expect.element('.js-test-location').attribute('data-location')
+    .to.contain(urlSubstring).before(Timeouts.slow);
+}
+
+// Expects navigation lands at a path with the given `urlSubstring`.
+function expectNavigateToExact(client, urlSubstring) {
```

Comment:
@cvalarida, @jbalboni, any preference regarding whether any of these new helpers should be added and others not?

Reply:
Yeah, these look good. My only suggestion is a slight rename:
- `expectNavigateTo` -> `expectLocation`
- `expectNavigateToExact` -> `expectExactLocation`

Reply:
Sounds good, changed.